cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -312,7 +312,7 @@ func (m *AwsManager) getAsgTemplate(asg *asg) (*asgTemplate, error) {
 	region := az[0 : len(az)-1]
 
 	if len(asg.AvailabilityZones) > 1 {
-		klog.Warningf("Found multiple availability zones for ASG %q; using %s\n", asg.Name, az)
+		klog.V(4).Infof("Found multiple availability zones for ASG %q; using %s for %s label\n", asg.Name, az, apiv1.LabelFailureDomainBetaZone)
```

Comment:
```suggestion
		klog.V(4).Infof("Found multiple availability zones for ASG %q; using %s for %s label\n", asg.Name, az, apiv1.LabelZoneFailureDomain)
```

We should stick to the same constant as is being used by the `buildGenericLabels` which is processing the AZ value returned by this function, though it will result in the same result in this case.

Is there a reason from moving from a warning to a high verbosity level Info message? I feel like this is generally an indicator to cluster operators that they'd want to be warned about.

Reply:
I agree but this warning is emitted each run and becomes an annoyance more than being useful.